client/src/extension.ts

```diff
@@ -46,6 +49,7 @@ export async function activate (context: vscode.ExtensionContext): Promise<void>
   bitbakeWorkspace.loadBitbakeWorkspace(context.workspaceState)
   bitbakeTaskProvider = new BitbakeTaskProvider(bitbakeDriver)
   client = await activateLanguageServer(context)
+  bitbakeRecipesView = new BitbakeRecipesView(context, bitbakeWorkspace)

```

Comment:
Seems like this instance of bitbakeRecipesView is not used anywhere. I know that some stuff including the view creation happens in the constructor. why don't we just declare a function that does that?

Reply:
I edited the code to use the variable in some way after the constructor.